docs/conf.py

```diff
@@ -61,6 +61,10 @@
     ("py:class", "SimTime"),
     ("py:class", "immutables._map.Map"),
     ("py:class", "_asyncio.Task"),
+    ("py:class", "tickit.devices.cryostream.status.Status"),
+    ("py:class", "tickit.devices.cryostream.status.ExtendedStatus"),
+    ("py:class", "optional"),
```

Comment:
Previously we've replaced auto-generated docstring args of form `some_arg (SomeType, optional)` to `some_arg (Optional[SomeType])`. But of-course these have two different meanings. I would suggest either changing to `Optional[SomeType]` here or if you're feeling brave you may want to change the rest of the docstrings to `SomeType, optional`.

Reply:
I'll take the path of least resistance.